repos/system_upgrade/el7toel8/actors/checksendmail/actor.py

```diff
@@ -0,0 +1,47 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.models import InstalledRedHatSignedRPM, SendmailMigrationDecision, TcpWrappersFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
```

Comment:
Please mind alphabetical order of imports https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md#5-avoid--imports-and-sort-imports-alphabetically

Reply:
It should be fixed by @examon commits.